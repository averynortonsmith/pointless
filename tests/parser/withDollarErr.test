
x = x with foo = bar