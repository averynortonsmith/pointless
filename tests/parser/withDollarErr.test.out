-------------------------------------------------------------------------------
Parser Error:

Expected 'Dollar', got 'Name'
-------------------------------------------------------------------------------

At (line 2 column 12) in 'tests/parser/withDollarErr.test'
x = x with foo = bar
           ^
